Blog post about commcall subtitles

Bisaloo opened the pull request:
* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags. (`accessibility` is a new tag)
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).

cc @Rekyt. 

We will probably still do one pass each but it's about ready for a first review and for your additions if you want @stefaniebutland

stefaniebutland commented:
Thank you!! I just glanced at the headings and I'm excited to read this. I will do a first review later today.

stefaniebutland commented:
...make that tomorrow

stefaniebutland commented on content/blog/2020-05-11-subtitles/index.md:
```diff
@@ -0,0 +1,126 @@
+Nonetheless, in case someone would like to try it next, we propose an optimised workflow, that may help reducing the time you spend:
+
+1. Use the transcript provided by Zoom/YouTube.
+1. Correct the beforehand transcript for typos and mistakes. In particular, pay attention to technical and field-specific terms (R, package names, URLs).
```
```suggestion
1. Correct the initial raw transcript for typos and mistakes. In particular, pay attention to technical and field-specific terms (R, package names, URLs).
```

stefaniebutland commented on content/blog/2020-05-11-subtitles/index.md:
```diff
@@ -0,0 +1,126 @@
+
+1. Upload the corrected transcript on Amara.
+1. Sync the subtitles with the audio. Because the breaks are already placed at
+the right time, this should be quick using amara and its keybord shortcuts, and drag and drop feature.
```
```suggestion
the right time, this should be quick using Amara and its keyboard shortcuts, and drag and drop feature.
1. Download the final transcript VTT file from Amara, upload it to the video on Vimeo, and enable Closed Captions to make them visible.
```

Bisaloo commented:
> Steffi LaZerte and Janani Ravi are preparing a post about the content of the community call itself so we can time these together.

@stefaniebutland, do you already know when you would like to time these two blog posts? It would probably help me to find the best time to fit this in my schedule.

stefaniebutland commented:
I've re-pinged Janani to establish a publication date. Will reply here

stefaniebutland commented:
Our plan is to publish your post on Wed July 15, so it would be ideal to have revisions by Tues July 7. The summary post, (pointing to specific spots in the video where questions were asked), will be published Tues July 14.

Bisaloo commented:
Thank you very much for your answer. We'll send the revisions as soon as possible, and for sure by Tues July 7!

stefaniebutland commented:
@Bisaloo @Rekyt A gentle reminder to make revisions by Tues July 7. I will review again after that.

stefaniebutland commented:
👋 When you're finished with edits, please tag @steffilazerte rOpenSci Community Intern. I've asked her to give her outsider perspective to make sure we're making sense 😉

Bisaloo commented on content/blog/2020-05-11-subtitles/index.md:
```diff
@@ -0,0 +1,126 @@
+
+#### Proposed optimised workflow
+
+Given the amount of time required to achieve a perfect result (or even a "good enough" result as in this case), we are not sure it worth continuing this enterprise, at least as long as we don't find better tools / methods.
```
I'm not sure it's really possible for us to answer this question. I think we would need to ask people who need the subtitles if (imperfect) auto-generated subtitles work for them or where exactly the threshold lies in their opinion.

Maybe it would be good to add a sentence mentioning this and suggesting that feedback on this specific point would be much appreciated.

Bisaloo commented:
I think Matthias did a good job addressing your comments. I can have a last read to try and improve the flow but it's getting very late so I won't be able to do it before tomorrow morning.

Feel free to have a look now if that's more convenient for you!

Rekyt commented:
👋 Hi @steffilazerte, I think I've answered most comments. The post is ready for review ;)
The post should probably be a little shorter to ease the reading. If you could point us to some passages that could be shortened, it would be really useful!

steffilazerte commented:
Awesome! I'll take a look either later today or first thing tomorrow!

Bisaloo commented:
I made some minor edits. Overall, I think many sentences are still way too long. This is a common issue when native French speakers write in English. Unfortunately, I didn't really manage to solve this issue entirely so we remain open to suggestions!
